pkg/secrets/cloud/manager_test.go

```diff
@@ -32,10 +32,10 @@ import (
 // the main testing function, takes a kms url and tries to make a new secret manager out of it and encrypt and
 // decrypt data, this is used by the aws_test and azure_test files.
 func testURL(ctx context.Context, t *testing.T, url string) {
-	dataKey, err := generateNewDataKey(url)
-	require.NoError(t, err)
+	info := &workspace.ProjectStack{}
```

Comment:
Again this change probably shouldn't be in this PR.

Reply:
I had updated this here already https://github.com/pulumi/pulumi/pull/14649/files#diff-418b37c4be779ba186bcac02d82a8c77335a15bcb58a9ee9d8169dbcb407a489R35 (in #14649), but the diff view didn't update, since I did that after pushing changes here.  This should be resolved once that PR is merged.